### PR TITLE
Add HoYoPlay Launcher

### PR DESCRIPTION
**Pull request by KotonBads**
Fixes #(issue)

## Type of change
- [x] New installer
- [ ] Manifest fix
- [ ] Other

# Whas This Tested Using a [Local Repository](https://maintainers.usebottles.com/Testing)?
- [x] Yes
- [ ] No

I'm not sure if this should just be an installer for HoYoPlay, which functions the same. The only difference I could see was the background images in the installer.

Icons taken from their respective websites, I don't have any higher res icons.

### CODE REVIEW

**Review by mirkobrombin — APPROVED** [review]

**Comment by mirkobrombin** [comment]
The changes looks good. Your guess is correct, at this point the installer for HoYoPlay would be better.

**Comment by KotonBads** [comment]
changed to hoyoplay launcher instead

File: Reviews/hoyoplay.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Zenless Zone Zero
```
Comment (mirkobrombin) [inline Reviews/hoyoplay.md:1]: I think this should be changed as well

Comment (mirkobrombin) [reply Reviews/hoyoplay.md:1]: and also the other occurrences

Comment (KotonBads) [reply Reviews/hoyoplay.md]: should be fixed now

**Pull request merged**